### PR TITLE
Fix descriptions of kubeadm

### PR DESCRIPTION
**Pull request by tnqn**
* The example iptables cmd only deleted chains of default table(filter)
* Fixed a typo in description of phase bootstrap-token

**What type of PR is this?**
/kind documentation

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: cmd/kubeadm/app/cmd/reset.go [hunk 1]
```diff
@@ -44,7 +44,7 @@ var (
 		The reset process does not reset or clean up iptables rules or IPVS tables.
 		If you wish to reset iptables, you must do so manually.
 		For example:
-		iptables -F && iptables -t nat -F && iptables -t mangle -F && iptables -X
+		iptables -F && iptables -t nat -F && iptables -t mangle -F && iptables -X && iptables -t nat -X && iptables -t mangle -X
```
Comment (neolit123) [inline cmd/kubeadm/app/cmd/reset.go]: @kubernetes/sig-cluster-lifecycle-pr-reviews 
PTAL

**Review by neolit123 — COMMENTED** [review]
thanks for the PR
/priority backlog
waiting on someone to double check the iptables change.

File: cmd/kubeadm/app/cmd/reset.go (hunk 1 above)
Comment (detiber) [reply cmd/kubeadm/app/cmd/reset.go]: I have some potential concerns that we may be telling users to kill rules that the user may be relying on, especially on local system installs, but that already exists with the existing command we are telling them to run.

Comment (neolit123) [reply cmd/kubeadm/app/cmd/reset.go]: i think we might want to remove the example and leave it to the users to figure things out by reading the docs for `iptables`.

proposing:
> If you wish to reset iptables, you must do so manually by using the `iptables` command.

Comment (dixudx) [reply cmd/kubeadm/app/cmd/reset.go]: > we might want to remove the example and leave it to the users to figure things out by reading the docs for iptables

Agree +1000!

Comment (neolit123) [reply cmd/kubeadm/app/cmd/reset.go]: @tnqn would you mind making the proposed change in https://github.com/kubernetes/kubernetes/pull/79648#discussion_r300121457
as part of this PR?

the idea is to more the iptables example completely and have that sentence instead.

Comment (tnqn) [reply cmd/kubeadm/app/cmd/reset.go]: Sure, done. Thanks for the suggestion.

**Review by neolit123 — COMMENTED** [review]
thanks
/ok-to-test
/lgtm
/approve

**Pull request merged**